.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+# Automatically perform a release whenever a new "release-like" tag is pushed to the repo.
+name: Release
+
+on:
+  push:
+    tags:
+      # Detect tags that look like a release.
+      # Note that we don't use a "v" prefix to help anchor this pattern.
+      # This is purely a matter of preference.
+      - "*.*.*"
+
+jobs:
+  release:
```

Comment:
rules_testing also populates release notes. https://github.com/bazelbuild/rules_testing/blob/master/.github/workflows/release.yml#L44

Do we want to do something similar?

I know this is more nice-to-have. But testing if this works is difficult, so maybe it's easier to add it now, so that we don't need to test twice.